src/x86_64-fma/fft16x16.py

```diff
@@ -499,6 +500,10 @@ def inverse_vfft(reg_t0, reg_t8, reg_t_stride, data_in, reg_row_start=None, reg_
                     negate_b=fft8_negate_b.get(id(data_hi), False),
                     writeback=False)
 
+            if relu:
+                ymm_zero = YMMRegister()
+                VMOVAPS(ymm_zero, Constant.uint32x8(0))
```

Comment:
This is for the backward pass. The backward pass needs to know which values were positive/negative before we applied ReLU. Using negative zero ensures that the sign of the convolution results doesn't change after we apply ReLU. See discussion in #24 for why its important.